bench-tps/src/cli.rs

```diff
@@ -106,6 +112,22 @@ pub fn build_args<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
                 .help("Per-thread-per-iteration sleep in ms"),
         )
+        .arg(
+            Arg::with_name("write-client-keys")
+                .short("w")
```

Comment:
IMO remove the short arg (w/r), the long arg is nicer

Reply:
👍